apps/studio/prisma/schema.prisma

```diff
@@ -33,6 +33,16 @@ model VerificationToken {
   expires    DateTime
 }
 
+model Version {
+  id          BigInt     @id @default(autoincrement())
+  versionNum  BigInt
+  resourceId  BigInt
+  resources   Resource[]
+  blobId      BigInt
+  blob        Blob       @relation(fields: [blobId], references: [id])
```

Comment:
question: should this be `@unique`?

Reply:
Each version should be tied to an unique blob ideally. But in case of a rollback, I am thinking:
- We create a new version pointing to the blobId referenced in the version we want to rollback to
- This is to prevent us from tampering old versions. Old versions should be immutable as they preserve things like publishedAt etc.
- This means 2 versions can point to the same blobId

Reply:
After offline discussions, it does make sense that we do not immediately create a version during rollbacks. We would likely clone the blobId of the version to rollback -> allow user to preview and then when they confirm to publish we will initiate a new version creation on this blob.

Hence, it would make sense to keep a unique blob to a version.